Assets/Plugins/Source/Editor/Utility/GUIEditorUtility.cs

```diff
@@ -22,10 +22,14 @@
 
 namespace PlayEveryWare.EpicOnlineServices.Editor.Utility
 {
+    using Microsoft.SqlServer.Server;
```

Comment:
What is this using for?

Reply:
You didn't know that the `GUIEditorUtility` is actually based on a massive `SQL` database? haha. No, silly auto using statement that my editor thought I needed based on something I errantly autocompleted from intellisense.

Reply:
I removed the using (along with a few others) that were unnecessary. Marking this conversation as resolved.